src/layouts/components/Header.tsx

```diff
@@ -243,28 +243,54 @@ function DesktopMenu({ item }: { item: string }) {
         setIsHoverOn!(true);
       }}
     >
-      <Menu.Button
-        onClick={() => {
-          setIsShowing(!isShowing);
-        }}
-        className={classNames(
-          {
-            "text-brand-100": isShowing && !isCursorOnHeader,
-          },
-          {
-            "!text-brand-100":
-              !isCursorOnHeader &&
-              !isHoverOn &&
-              router.pathname.includes(item.toLowerCase()),
-          }
-        )}
-      >
-        <div className={classNames("flex flex-col")}>
-          {(item === MobileTabletDropDownState.ECOSYSTEM ||
-            item === MobileTabletDropDownState.BUILD) && <ComingSoonTag />}
-          {t(`header.navbar.${item.toLowerCase()}`)}
-        </div>
-      </Menu.Button>
+      {isSafariBelow164() ? (
+        <Menu.Button
+          onClick={() => {
+            setIsShowing(!isShowing);
+          }}
+          className={classNames(
+            isHoverOn && isShowing ? "text-brand-100" : "text-dark-700",
```

Comment:
From what i see, the only difference on the two condition is this line. Does this has anything to do with assigning linear-gradient to background image like what you described in your PR?

Reply:
if the only diff is just this conditional styling, we dont have to use the conditional operator on the whole `<Menu.Button>`, just the styling enough

Reply:
for this `isHoverOn && isShowing ? "text-brand-100" : "text-dark-700",` if its set here then it will overwrite the linear-gradient color set onMouseLeave and set text-dark-700 on the `MenuButton`